discovery-provider/scripts/test.sh

```diff
@@ -17,7 +17,7 @@ set -e
 # rm -r venv
 # python3 -m venv venv
 # source venv/bin/activate
-pip3 install -r requirements.txt
+python3.9 -m pip install -r requirements.txt
```

Comment:
Thanks for this! Do we also need to update `docker-compose` => `docker compose`?

Reply:
this file can actually be deleted after this PR, but happy to make that change here anyway